src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/Rules/PropertyPages/BuildPropertyPage.xaml

```diff
@@ -145,7 +145,7 @@
   <!-- TODO create fwlink for this HelpUrl -->
   <BoolProperty Name="CheckForOverflowUnderflow"
                 DisplayName="Check for arithmetic overflow"
-                Description="Specifies whether integer arithmetic that results in a value outside the range of the data type, and that is not in the scope of a checked or unchecked keyword, causes a run-time exception."
+                Description="Integer arithmetic that results in an out of range value causes a run-time exception."
```

Comment:
Doesn't use an action verb at the start... Flipping the sentence around would do it, and replace 'cause' with 'throw':
```suggestion
                Description="Throw a run-time exception when integer arithmetic that results in an out-of-range value occurs."
```